content/components/tooltip.mdx

```diff
@@ -3,42 +3,239 @@ title: Tooltip
-- Primer::Tooltip
+  - Primer::Alpha::Tooltip
+  - Primer::Tooltip
 cssId: tooltip
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Link, Text} from '@primer/react'
+import {Link, Text, Box} from '@primer/react'
 import {Link as GatsbyLink} from 'gatsby'
 export default ComponentLayout
 
+<Note variant="danger">
+  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>⚠️ Usage Warning ⚠️</Text>
+  <Text>
+    Tooltips as a UI pattern should be the last resort for conveying information as they are hidden by default and often
+    with zero or little visual indicator of its existence. See tooltip alternaitves below before using this component.
+  </Text>
+</Note>
 
 ## Usage
 
-The tooltip component adds a tooltip to add context to interactive elements on the page.
-
 <img
   width="960"
   role="presentation"
-  src="https://github.com/primer/design/assets/586552/787873b7-d496-454c-b669-0c2400ef503f"
+  src="https://github.com/primer/primitives/assets/18661030/172aa8f6-2273-4719-bb63-8dd678ffcbc9"
   alt="three examples of tooltips on interactive elements"
 />
 
-## Accessibility
+- Reserve tooltips for components such as icon buttons.
+- Keep the tooltip text minimal.
+- **Never** include tooltips on non-interactive components (`div`, `span`, `p`).
 
-<Note variant="danger">
-  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>⚠️ Usage Warning ⚠️</Text>
-  <Text>Tooltips as a UI pattern should be our last resort for conveying information because it is hidden by default and often with zero or little visual indicator of its existence.</Text>
-</Note>
+### Best practices
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      alt="Icon buttons with tooltips for edit, favorite, and sponsor"
+      src="https://github.com/primer/primitives/assets/18661030/e9f86af8-3792-4794-af68-ad0efccd8bc8"
+    />
+    <Caption>Use a tooltip to give a text label to an icon button</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      alt="Non-interactive text element activating a long and wordy tooltip on hover"
+      src="https://github.com/primer/primitives/assets/18661030/1d8d86a2-8ee7-40d2-8934-c3d36f34d9b3"
+    />
+    <Caption>Don't use tooltips on non-interactive elements</Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="800"
+      alt=""
+      src="https://github.com/primer/primitives/assets/18661030/d5528bb1-cf2e-41f6-b029-d861fb28375b"
+    />
+    <Caption>Do use tooltips for icon buttons that serve unique purposes</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="800"
+      alt=""
+      src="https://github.com/primer/primitives/assets/18661030/9713ad8b-9f52-44fd-8b46-06d5151b813f"
+    />
+    <Caption>
+      Don’t use tooltips for icon buttons that appear in a group of other controls which share the same functionality,
+      such as opening a context menu.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Label
```

Comment:
Could we mention how a label type tooltip should be reserved for icon buttons? The underlying semantic of a label tooltip is `aria-labelledby` would override any label the button already has.

Reply:
I feel like this is clear already by saying "For interactive controls with no visible text label" but I will add "such as icon buttons" 😄